packages/cli/src/Server.ts

```diff
@@ -1656,16 +1677,16 @@ class App {
 				.take(limit);
 
 			Object.keys(filter).forEach((filterField) => {
-				resultsQuery.andWhere(`execution.${filterField} = :${filterField}`, {[filterField]: filter[filterField]});
+				resultsQuery.andWhere(`execution.${filterField} = :${filterField}`, { [filterField]: filter[filterField] });
```

Comment:
Worth checking if quotes will be needed in these query bits since (I think) Postgres auto-lowercases non-quoted strings.